composer.json

```diff
@@ -23,23 +23,22 @@
   },
   "require": {
     "billpay/lib": "1.6.0",
-    "spryker/application": "^2.4.4",
-    "spryker/config": "^2.0.0",
-    "spryker/country": "^2.0.0",
-    "spryker/currency": "^1.0.0",
-    "spryker/gui": "^2.0.0",
-    "spryker/kernel": "^2.0.0",
+    "spryker/application": "^3.0.0",
```

Comment:
Do you really require the latest versions only? Doesn't it work with ^2.4.4?
Others?